code/go/0chain.net/smartcontract/storagesc/models.go

```diff
@@ -1707,63 +1567,57 @@ func (sa *StorageAllocation) validateEachBlobber(
 	)
 	for i, b := range blobbers {
 		sn := StorageNode{}
-		beforeArtemisFork := func() error {
-			sn1 := storageNodeResponseToStorageNodeV1(*b)
-			sn.SetEntity(sn1)
-			err := sa.isActive(&sn, b.TotalStake, b.TotalOffers, b.StakedCapacity, conf, creationDate)
-			if err != nil {
-				logging.Logger.Debug("error validating blobber", zap.String("id", b.ID), zap.Error(err))
-				return err
-			}
-			return nil
-		}
 
-		artemisFork := func() error {
+		beforeHardfork := func() error {
 			snr := storageNodeResponseToStorageNodeV2(*b)
 			sn.SetEntity(snr)
-			if *snr.IsRestricted {
-				success, err := verifyBlobberAuthTicket(balances, sa.Owner, blobberAuthTickets[i], snr.PublicKey)
-				if err != nil {
-					return fmt.Errorf("blobber %s auth ticket verification failed: %v", b.ID, err.Error())
-				} else if !success {
-					return fmt.Errorf("blobber %s auth ticket verification failed", b.ID)
-				}
-			}
-
 			return nil
```

Comment:
Should we remove the snr.IsRestricted process logic? I see it processed in both artemisFork and athenaFork, if we remove it, directly, that means we are changing logic in both before and after activation, which can cause different state.